scripts/benchmarks/run_benchmarks.sh

```diff
@@ -7,6 +7,10 @@ set -eou pipefail
 
 PROFILE="${PROFILE:-production}"
 
+EXECUTION="${EXECUTION:-wasm}"
+
+ADDITIONAL="${ADDITIONAL:-}"
+
```

Comment:
I think this part logically belongs into *scripts/benchmarks/configuration.sh*.

Reply:
It does indeed! c96054002660b666b62e4bb7c3f5f02264664b4c